java/mtsj/core/src/main/java/com/devonfw/application/mtsj/general/service/impl/config/BaseWebSecurityConfig.java

```diff
@@ -88,11 +89,11 @@ public void configure(HttpSecurity http) throws Exception {
   @SuppressWarnings("javadoc")
   public void configure(AuthenticationManagerBuilder auth) throws Exception {
 
-    auth.inMemoryAuthentication().withUser("waiter").password(this.passwordEncoder.encode("waiter")).roles("Waiter")
-        .and().withUser("cook").password(this.passwordEncoder.encode("cook")).roles("Cook").and().withUser("barkeeper")
-        .password(this.passwordEncoder.encode("barkeeper")).roles("Barkeeper").and().withUser("chief")
-        .password(this.passwordEncoder.encode("chief")).roles("Chief").and()
-        .withUser("manager").password(this.passwordEncoder.encode("manager")).roles("Manager");
+    DaoAuthenticationProvider authProvider = new DaoAuthenticationProvider();
+    authProvider.setUserDetailsService(this.userDetailsService);
+    authProvider.setPasswordEncoder(this.passwordEncoder);
+    auth.authenticationProvider(authProvider);
```

Comment:
Great that we went away from this in memory authentication hack. Using a real AuthenticationProvider and having the accounts with PWD not redundant is a great improvement.